src/v2/view-builder/views/duo/ChallengeDuoAuthenticatorView.js

```diff
@@ -0,0 +1,35 @@
+import { loc } from 'okta';
+import Duo from 'duo';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+
+const Body = BaseForm.extend({
+
+  className: 'duo-authenticator-challenge',
+  noButtonBar: true,
+
+  title  () {
+    return loc('oie.duo.verify.title', 'login');
+  },
+
+  postRender: function () {
+    const contextualData = this.options.appState.get('currentAuthenticator').contextualData;
+    // TODO Check contextualData.integrationType once we support other types than iframe
+    this.add('<iframe frameborder="0" title="' + this.title() + '"></iframe>');
```

Comment:
style-nit: Template literals are easier to read!

```suggestion
    this.add(`<iframe frameborder="0" title="'${this.title()}'"></iframe>`);
```